observability/logging/logging-6.0/log6x-upgrading-to-6.adoc

```diff
@@ -0,0 +1,433 @@
+
+[IMPORTANT]
+====
+To continue using an existing Red Hat managed Elasticsearch or Kibana deployment provided by the *elasticsearch-operator*, remove the owner references from the `Elasticsearch` resource named `elasticsearch`, and  the `Kibana` resource named `kibana` in the `openshift-logging` namespace before removing the `ClusterLogging` resource named `instance` in the same namespace.
+====
+
+
+. Temporarily set *ClusterLogging* to state `Unmanaged`
++
+[source,terminal]
+----
+$ oc -n openshift-logging patch clusterlogging/instance -p '{"spec":{"managementState": "Unmanaged"}}' --type=merge
+----
+
+. Remove *ClusterLogging* `ownerReferences` from the *Elasticsearch* resource
++
+The following command ensures that *ClusterLogging* no longer owns the *Elasticsearch* resource. Updates to the *ClusterLogging* resource's `logStore` field will no longer affect the *Elasticsearch* resource.
++
+[source,terminal]
+----
+$ oc -n openshift-logging patch elasticsearch/elasticsearch -p '{"metadata":{"ownerReferences": []}}' --type=merge
+----
+
+. Remove *ClusterLogging* `ownerReferences` from the *Kibana* resource
++
+The following command ensures that *ClusterLogging* no longer owns the *Kibana* resource. Updates to the *ClusterLogging* resource's `visualization` field will no longer affect the *Kibana* resource.
++
+[source,terminal]
+----
+$ oc -n openshift-logging patch kibana/kibana -p '{"metadata":{"ownerReferences": []}}' --type=merge
+----
+
+. Set *ClusterLogging* to state `Managed`
+[source,terminal]
+----
+$ oc -n openshift-logging patch clusterlogging/instance -p '{"spec":{"managementState": "Managed"}}' --type=merge
+----
+
+== Log Visualization
+[subs="+quotes"]
+The OpenShift console UI plugin for log visualization has been moved to the *cluster-observability-operator* from the *cluster-logging-operator*.
+// Pending support statement.
+
+
+
+== Log Collection and Forwarding
+// Can't link to github, need to figure a workaround.
+
+Log collection and forwarding configurations are now specified under the new link:https://github.com/openshift/cluster-logging-operator/blob/master/docs/reference/operator/api_observability_v1.adoc[API], part of the `observability.openshift.io` API group. The following sections highlight the differences from the old API resources.
+
+[NOTE]
+====
+Vector is the only supported collector implementation.
+====
+
+== Management, Resource Allocation, and Workload Scheduling
+
+Configuration for management state (e.g., Managed, Unmanaged), resource requests and limits, tolerations, and node selection is now part of the new *ClusterLogForwarder* API.
+
+.Previous Configuration
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+spec:
+  managementState: "Managed"
```

Comment:
I think we should standardize on the "non-quoted" form for where it's not needed:

```suggestion
apiVersion: logging.openshift.io/v1
kind: ClusterLogging
spec:
  managementState: Managed
```

This applies at least to all mentions of `apiVersion` and `kind`, but I don't think it makes sense to pollute the PR with comments on each.